content/docs/command-reference/add.md

```diff
@@ -148,15 +165,29 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
-- `--desc <text>` - user description of the data (optional). This doesn't affect
+- `--to-remote` - transfer data straight to remote, when the used system doesn't
+  have the means to store it locally. So instead of transferring it to the local
+  cache and link it to the working directory, it is transferred through the
+  local computer in batches to the remote storage (can be configured using
+  `--remote <name>`) and can be checked out locally when the necessary means
+  have been established since this process also results with a DVC file.
+
+* `-o <filename>`, `--out <filename>` - destination path for the transferred
+  data. (Can only be used with `--to-remote`)
+
+* `-r <name>`, `--remote <name>` - name of the
+  [remote storage](/doc/command-reference/remote). (Can only be used with
+  `--to-remote`)
+
+* `--desc <text>` - user description of the data (optional). This doesn't affect
   any DVC operations.
 
-- `-h`, `--help` - prints the usage/help message, and exit.
+* `-h`, `--help` - prints the usage/help message, and exit.
```

Comment:
not related? also, pre-commit should be enforcing `-` style

Reply:
Probably @isidentical's IDE is doing some auto-formatting, please check out https://dvc.org/doc/user-guide/contributing/docs 🙂

Reply:
hmm weird, perhaps I use the wrong thing somewhere and the autoformatter started replacing others that are below my misusage. should be fixed now!